src/ManagedServiceIdentity/docs/Update-AzUserAssignedIdentity.md

```diff
@@ -0,0 +1,222 @@
+---
+external help file:
+Module Name: Az.ManagedServiceIdentity
+online version: https://docs.microsoft.com/powershell/module/az.managedserviceidentity/update-azuserassignedidentity
+schema: 2.0.0
+---
+
+# Update-AzUserAssignedIdentity
+
+## SYNOPSIS
+Update an identity in the specified subscription and resource group.
+
+## SYNTAX
+
+### UpdateExpanded (Default)
+```
+Update-AzUserAssignedIdentity -Name <String> -ResourceGroupName <String> [-SubscriptionId <String>]
+ [-Location <String>] [-Tag <Hashtable>] [-DefaultProfile <PSObject>] [-Confirm] [-WhatIf]
```

Comment:
I suspect Location is required for `Update-AzUserAssignedIdentity`, could you verify it?

Reply:
I have verified that the Location is optional for update operation.